zipkin-cassandra-core/src/main/java/org/twitter/zipkin/storage/cassandra/Repository.java

```diff
@@ -70,6 +70,23 @@
     private final PreparedStatement selectTraceDurationTail;
     private final Map<String,String> metadata;
 
+    private final ThreadLocal<Set<String>> writtenNames = new ThreadLocal<Set<String>>() {
+                private long hour = TimeUnit.MILLISECONDS.toHours(System.currentTimeMillis());
```

Comment:
Fwiw I think most of the span stores accept a TTL, which not sure if the
granularity is right. That said, James raises a good point. We don't have a
dynamic config story. At least having it statically configured (similar to
default ttl in SpanStore) would seem a good start.